.github/workflows/update-jira-fix-version.yml

```
@@ -0,0 +1,302 @@
+
+  # ---------------------------------------------------------------------------
+  # Job 2: Automated cherry-pick PR merged → set Fix Version to CURRENT release.
+  #
+  # Cherry-pick PRs have no MM ticket key, so Jira automation cannot link them.
+  # This job traces back to the original PR to find the ticket key.
+  #
+  # Cherry-pick title format: "Automated cherry pick of #37124"
+  # ---------------------------------------------------------------------------
+  cherry-pick-pr:
+    name: Cherry-pick PR → current release
+    if: >
```

actionlint (1.7.12)

[error] undefined function "toLower". available functions are "always", "cancelled", "case", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)